### ISSUE TITLE
Better access to trial logs

### ISSUE THREAD

**Issue by XiaoXiao-Woo** [microsoft/nni#4523]
**What would you like to be added**:
Could you support better access to trial logs?

**Why is this needed**:
User interaction experience is bad. 

**Without this feature, how does current nni work**：
 The old command "nnictl view [id] -p [port]" need to dispatch multiple port for viewing multiple trial id.
"nnictl view hfopmcn3; nnictl view k7ejanqp", the latter failed but "nnictl view k7ejanqp -p 8081" passed.

**Components that may involve changes**:
better access to trial logs

**Brief description of your proposal if any**:
1. Webui could view or click directly other trial ids in a port.
2. nnictl view -h. It has no requirement for id. 
![image](https://user-images.githubusercontent.com/15083102/151693646-f86527eb-d0c3-47cc-82b4-205fabbc0c6b.png)

**Comment by QuanluZhang** [comment]
@XiaoXiao-Woo thanks for reporting this feature request. The figure you provided is the UI of experiment management, each ID represents an experiment not trial. Could you describe more about the feature you want. You mean you want to view different experiments using the same port? or you want to view the experiment with a newly specified port? or you actually want to view the trials' log of an experiment?

**Comment by XiaoXiao-Woo** [comment]
Yes, I hope different experiments could use the same port so that experiment ids could be dispatched with different url(router) to access them when click experiment id list.  I think that's more convenience.

**Comment by QuanluZhang** [comment]
we will consider this feature request, and post design proposal when we are ready.
